src/test/ui/impl-trait/multiple-lifetimes/ordinary-bounds-unrelated.stderr

```diff
@@ -4,11 +4,7 @@ error[E0700]: hidden type for `impl Trait` captures lifetime that does not appea
 LL | fn upper_bounds<'a, 'b, 'c, 'd, 'e>(a: Ordinary<'a>, b: Ordinary<'b>) -> impl Trait<'d, 'e>
    |                                                                          ^^^^^^^^^^^^^^^^^^
    |
-note: hidden type `Ordinary<'_>` captures lifetime smaller than the function body
-  --> $DIR/ordinary-bounds-unrelated.rs:16:74
-   |
-LL | fn upper_bounds<'a, 'b, 'c, 'd, 'e>(a: Ordinary<'a>, b: Ordinary<'b>) -> impl Trait<'d, 'e>
-   |                                                                          ^^^^^^^^^^^^^^^^^^
+   = note: hidden type `Ordinary<'_>` captures lifetime '_#9r
```

Comment:
I'll make a separate PR that fixes this diagnostic just for nll. We should probably not merge this PR before that not-yet-written nll diagnostics PR

Reply:
This PR is up now: https://github.com/rust-lang/rust/pull/89327